backend/src/main/java/ai/verta/modeldb/entities/versioning/RepositoryEntity.java

```diff
@@ -231,7 +239,6 @@ private void updateAttribute(List<KeyValue> attributes) throws InvalidProtocolBu
   }
 
   public boolean isDataset() {
-    // TODO: keep track if it is a dataset https://vertaai.atlassian.net/browse/VR-4571
-    return true;
+    return datasetRepositoryMappingEntity.getIsDatasetRepository();
```

Comment:
return datasetRepositoryMappingEntity!= null 

would this work?

Reply:
Yes, it works. I have changed it as per your suggestion